CMakeLists.txt

```diff
@@ -918,8 +918,8 @@ if(CMAKE_COMPILER_IS_GNUCXX AND HPX_WITH_GCC_VERSION_CHECK)
 endif()
 
 if(MSVC)
-  if(NOT (MSVC11 OR MSVC12 OR MSVC14))
-    hpx_error("MSVC x64 2012 or higher is required.")
+  if(NOT (MSVC14))
```

Comment:
Yes it's definitly out of scope, I just wanted to note it so we can pick it up in the future...